apps/prairielearn/src/cron/autoFinishExams.ts

```diff
@@ -16,29 +19,33 @@ import { config } from '../lib/config';
  * @see assessment.gradeAssessmentInstance
  */
 export async function run() {
-  const params = [config.autoFinishAgeMins];
-  const result = await sqldb.callAsync('assessment_instances_select_for_auto_finish', params);
-  const examList = result.rows;
+  const examList = await sqldb.queryRows(
+    sql.select_assessments_to_auto_close,
+    { age_minutes: config.autoFinishAgeMins },
+    AssessmentInstanceSchema.pick({ id: true, open: true, assessment_id: true }),
+  );
 
   await async.eachSeries(examList, async (examItem) => {
```

Comment:
Nit, we could use a normal `for ... of ...` loop now that we're not using callbacks.